intel_extension_for_transformers/neural_chat/tests/ci/api/test_chatbot_build_api.py

```diff
@@ -30,24 +30,24 @@ def tearDown(self) -> None:
         return super().tearDown()
 
     def test_build_chatbot_with_default_config(self):
-        config = PipelineConfig(model_name_or_path="facebook/opt-125m")
+        config = PipelineConfig(model_name_or_path="/tf_dataset2/models/nlp_toolkit/opt-125m")
         chatbot = build_chatbot(config)
         self.assertIsNotNone(chatbot)
         response = chatbot.predict(query="Tell me about Intel Xeon Scalable Processors.")
         print(response)
         self.assertIsNotNone(response)
 
     def test_build_chatbot_with_customized_pipelinecfg(self):
-        config = PipelineConfig(model_name_or_path="facebook/opt-125m",
-                                tokenizer_name_or_path="EleutherAI/gpt-neox-20b")
+        config = PipelineConfig(model_name_or_path="/tf_dataset2/models/nlp_toolkit/opt-125m",
```

Comment:
suggest not using specific path e.g., tf_datasets?

Reply:
@VincyZhang please check haihao's comments.

Reply:
In order to save test time, we use local model. It's just a folder name created inside the test docker, we can also change to `/datasets`.